packages/node_modules/@ciscospark/internal-plugin-conversation/src/decryption-transforms.js

```diff
@@ -70,6 +70,10 @@ export const transforms = toArray('inbound', {
     return ctx.transform('decryptObject', keyUrl, activity.object);
   },
 
+  decryptMicroappinstance(ctx, key, microappInstance) {
```

Comment:
`decryptMicroappInstance(ctx, key, microappInstance)`

Reply:
The capitalize function inside of `decryptOobject` turns `microappInstance` into `decryptMicroappinstance`. It definitely bothers me, too :) I guess we could switch that to a camel-case function instead of capitalize?